Update the changelog for 1.3.1 (#1419)
astronomer/astro-sdk: python-sdk/docs/CHANGELOG.md

# Changelog

## 1.3.1

### Feature:

> - Introduce a parameter `if_exists` which user can set to `append` or `replace` for `aql.dataframe()` operator. `append` insert into an existing table (`output_table`) from a Dataframe. [#1379](https://github.com/astronomer/astro-sdk/issues/1379)

Reviewer: ```suggestion
- `aql.dataframe()` operator now supports `append` mode in addition to the default of `replace`. `append` inserts records into an existing table (`output_table`) from the dataframe. [#1379](https://github.com/astronomer/astro-sdk/issues/1379)
```

> - Temporary Schemas accounts for Location/Region when creating BigQuery schema. User can now create schema in Bigquery as per region like - `<dataset_id>__<region_id>`. [#1369](https://github.com/astronomer/astro-sdk/issues/1369)

Reviewer: Would have been good to add a quick example. Have we covered this case in docs too? I don't see anything in https://github.com/astronomer/astro-sdk/pull/1416/files . How would a user know about this feature?

cc @phanikumv @utkarsharma2

### Bug fixes

> - Fix the `aql.cleanup()` operator as failing as the attribute `output` was implemented in 2.4.0 [#1359](https://github.com/astronomer/astro-sdk/issues/1359)

Reviewer: This comment does not give much info from a User's perspective as they don't know what `output` attribute is?

It should be the following instead:

```suggestion
- Make `aql.cleanup()` operator compatible with Task Mapping features introduced in Airflow 2.4.0 [#1359](https://github.com/astronomer/astro-sdk/issues/1359)
```

- Fix the backward compatibility with `apache-airflow-providers-snowflake==4.0.2`. [#1351](https://github.com/astronomer/astro-sdk/issues/1351)

> - LoadFile operator returns a dataframe if not using XCom backend.[#1348](https://github.com/astronomer/astro-sdk/pull/1348),[#1337](https://github.com/astronomer/astro-sdk/issues/1337)

Reviewer: Some of these points end with a "dot" while the others don't, I am happy with either but let's be consistent

Reviewer: ```suggestion
- Fix backward compatibility with `apache-airflow-providers-snowflake==4.0.2`. [#1351](https://github.com/astronomer/astro-sdk/issues/1351)
- `LoadFile` operator returns a dataframe when not using XCom backend shipped with Astro SDK [#1348](https://github.com/astronomer/astro-sdk/pull/1348),[#1337](https://github.com/astronomer/astro-sdk/issues/1337)
```
